zerver/tests/test_signup.py

```diff
@@ -331,10 +330,10 @@ def test_password_reset_for_deactivated_user(self) -> None:
 
         # Check that the password reset email is from a noreply address.
         from django.core.mail import outbox
-        from_email = outbox[0].from_email
-        self.assertIn("Zulip Account Security", from_email)
-        tokenized_no_reply_email = parseaddr(from_email)[1]
-        self.assertTrue(re.search(self.TOKENIZED_NOREPLY_REGEX, tokenized_no_reply_email))
+        self.assertRegex(
+            outbox[0].from_email,
+            fr"^Zulip Account Security <{self.TOKENIZED_NOREPLY_REGEX}>\Z",
+        )
```

Comment:
Maybe we should extract this to a test helper we can call on `.from_email`?  It's kinda a mouthful.